src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -893,25 +893,28 @@ final case class EServicesApiServiceImpl(
           "voucherLifespan"         -> Json.fromInt(descriptor.voucherLifespan),
           "dailyCallsPerConsumer"   -> Json.fromInt(descriptor.dailyCallsPerConsumer),
           "dailyCallsTotal"         -> Json.fromInt(descriptor.dailyCallsTotal),
-          "description"             -> descriptor.description.map(Json.fromString).getOrElse(Json.obj()),
-          "agreementApprovalPolicy" -> Json.fromString(descriptor.agreementApprovalPolicy.toString)
+          "description"             -> descriptor.description.map(Json.fromString).getOrElse(Json.Null),
```

Comment:
same

Reply:
[Same](https://github.com/pagopa/interop-be-backend-for-frontend/pull/289/files#r1660596865)